### DISCUSSION TITLE
Limit rowlist component options in danger

### DISCUSSION THREAD

**Discussion by paulasuarezb** [Telefonica/mistica-design#1753] [Proposals]
### Goal

Rowlist danger may be used with onpress + href only

### Context

Due to the nature of the component, it will be possible to enable a rowlist in danger with checkbox, switch and other presets that may not really have a use. Limit the options so that in this state it works with only onpress and hfref (or other cases where it makes sense).

### Impact

The options will be limited in this danger state of the component.

### Discussion Questions

- what pressets should be limited?

**Comment by yceballost** [comment]
We will check the analytics of the libraries and decide later on
